spec/book_spec.rb

```diff
@@ -0,0 +1,14 @@
+require_relative '../book'
+
+describe Book do
+  before do
+    @book = Book.new('The Great Gatsby', 'F. Scott Fitzgerald')
+  end
+
+  describe '#initialize' do
+    it 'initializes with a title and author' do
+      expect(@book.title).to eq('The Great Gatsby')
+      expect(@book.author).to eq('F. Scott Fitzgerald')
+    end
+  end
+end
```

Comment:
- [ ] Great job so far 👏🏻 👏🏻 However, according to the project requirements, there should be unit tests for all the methods in all the classes, Kindly make sure the `add_rental` classbook method has (at least) an 'it' block 👇🏻 check your book.rb file

```ruby
  def add_rental(rental)
    @rentals.push(rental) unless @rentals.include?(rental)
    rental.book = self
  end
```